defaults/main.yml

```diff
@@ -2,6 +2,8 @@
 
 # --- Defaults (Welcome to override!) ---
 packagecontrol_url: "https://packagecontrol.io/Package Control.sublime-package"
+packagecontrol_dir_bin: "~/.config/sublime-text-3/Installed\ Packages"
```

Comment:
Good catch. See https://github.com/cmprescott/ansible-role-sublimetext3_packagecontrol/commit/f8f935219dfe7c2d16f90a5aff742fc6a5e7b229 

This was just one of the things I tried to get Travis to complete the build. However, it is not needed.